cmd/up.go

```diff
@@ -200,10 +200,6 @@ func ClusterUp(ctx context.Context, dialersOptions hosts.DialersOptions, flags c
 		return APIURL, caCrt, clientCert, clientKey, nil, err
 	}
 
-	if err := cluster.RestartKubeAPIServerWhenConfigChanges(ctx, kubeCluster, currentCluster); err != nil {
```

Comment:
@jiaqiluo The main reason for just one api server to restart is cuz we return here, right? So wouldn't it work if we restarted kube api for all hosts here? Or the problem is only DeepEqual comparison isn't sufficient to decide when config changes?

Reply:
@kinarashah,  this approach breaks the rolling upgrade (or zero-downtime upgrade) for control plane nodes: it will restart the kube-apiserver container on all CP nodes at once( if I remove the return). That is the main reason why I decided to use the "adding an env var to kube-apiserver's docker configuration" approach.